client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java

```diff
@@ -1858,6 +1859,58 @@ public void testVerifyRepository() {
         assertThat(expectedParams, equalTo(request.getParameters()));
     }
 
+    public void testGetSnapshots() {
+        Map<String, String> expectedParams = new HashMap<>();
+        String repository = randomIndicesNames(1, 1)[0];
+        String snapshot1 = "snapshot1-" + randomAlphaOfLengthBetween(2, 5).toLowerCase(Locale.ROOT);
+        String snapshot2 = "snapshot2-" + randomAlphaOfLengthBetween(2, 5).toLowerCase(Locale.ROOT);
+
+        String endpoint = String.format(Locale.ROOT, "/_snapshot/%s/%s,%s", repository, snapshot1, snapshot2);
+
+        GetSnapshotsRequest getSnapshotsRequest = new GetSnapshotsRequest();
+        getSnapshotsRequest.repository(repository);
+        getSnapshotsRequest.snapshots(Arrays.asList(snapshot1, snapshot2).toArray(new String[0]));
+        setRandomMasterTimeout(getSnapshotsRequest, expectedParams);
+        if (randomBoolean()) {
+            getSnapshotsRequest.ignoreUnavailable(true);
+            expectedParams.put("ignore_unavailable", Boolean.TRUE.toString());
+        }
+        if (randomBoolean() == false) {
+            getSnapshotsRequest.verbose(false);
+            expectedParams.put("verbose", Boolean.FALSE.toString());
+        }
+
+        Request request = RequestConverters.getSnapshots(getSnapshotsRequest);
+        assertThat(endpoint, equalTo(request.getEndpoint()));
+        assertThat(HttpGet.METHOD_NAME, equalTo(request.getMethod()));
+        assertThat(expectedParams, equalTo(request.getParameters()));
+        assertNull(request.getEntity());
+    }
+
+    public void testGetAllSnapshots() {
+        Map<String, String> expectedParams = new HashMap<>();
+        String repository = randomIndicesNames(1, 1)[0];
```

Comment:
Why not `randomAlphaOfLength(5)` or something?

Reply:
That is just how all tests in the class does it? It labels it as an index in the name. Maybe for debugging?

Reply:
Weird. When in Rome, I guess.

Reply:
Its cuz when i did it, i had no idea that other thing existed. Would be a nice fixup PR after all these Snapshots related PRs got finished.